src/deroff.rs

```diff
@@ -538,12 +538,311 @@ impl Deroffer {
         unimplemented!()
     }
 
+    fn quoted_arg(&self) -> bool {
+        unimplemented!()
+    }
+
+    fn request_or_macro(&mut self) -> bool {
+        // self.skip_char();
+        self.s = self.skip_char(&self.s, None).to_owned();
+
+        let s0 = &self.s[1..2];
+
+        match s0 {
+            "\\" => {
+                // if self.str_at(1) == Some('"') {
+                if Self::str_at(&self.s, 1) == "\"" {
+                    // self.condputs("\n");
+                    return true;
+                }
+            }
+            "[" => {
+                self.refer = true;
+                // self.condputs("\n");
+                return true;
+            }
+            "]" => {
+                self.refer = false;
+                // self.skip_char();
+                self.s = self.skip_char(&self.s, None).to_owned();
+                // return self.text();
+            }
+            "." => {
+                self.r#macro = false;
+                // self.condputs("\n");
+                return true;
+            }
+            _ => (),
+        };
+
+        self.nobody = false;
+        let s0s1 = self.s[..2].to_owned();
+
+        if self.g_macro_dispatch(s0s1.as_str()) {
+            return true;
+        }
+
+        if SKIP_HEADERS && self.nobody {
+            return true;
+        }
+
+        self.s = self.skip_leading_whitespace(&self.s).to_owned();
+        while !self.s.is_empty() && !Self::is_white(&self.s, 0) {
+            self.s = self.skip_char(&self.s, None).to_owned();
+        }
+
+        self.s = self.skip_leading_whitespace(&self.s).to_owned();
+
+        loop {
+            if !self.quoted_arg() && !self.text_arg() {
+                if !self.s.is_empty() {
+                    // self.condputs(self.str_at(0));
+                    self.condputs(Self::str_at(&self.s, 0));
+                    self.s = self.skip_char(&self.s, None).to_owned();
+                } else {
+                    return true;
+                }
+            }
+        }
+    }
+
+    fn font(&mut self) -> bool {
+        if let Some(m) = self.g_re_font.find(&self.s) {
+            self.s = self.skip_char(&self.s, Some(m.end())).to_owned();
+            true
+        } else {
+            false
+        }
+    }
+
+    fn comment(&self) -> bool {
+        unimplemented!()
+    }
```

Comment:
I believe comment is already implemented right? Maybe you might need to rebase this with master?
https://github.com/rust-dc/fish-manpage-completions/blob/2e8c847b2bf0f04556126668aeef837425c34506/src/deroff.rs#L369-L374

Reply:
fixed